src/main/java/com/ssafy/ssafybox/model/dao/SsafyboxDaoImpl.java

```diff
@@ -0,0 +1,117 @@
+package com.ssafy.ssafybox.model.dao;
+
+import com.ssafy.ssafybox.model.dto.TodoDto;
+import org.springframework.stereotype.Repository;
+import com.ssafy.ssafybox.util.DBUtil;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class SsafyboxDaoImpl implements SsafyboxDao {
+
+    public boolean createTodo(String content) throws SQLException {
+        Connection c = null;
+        PreparedStatement p = null;
+
+        try {
+            c = DBUtil.getConnection();
+            String query = "INSERT INTO todos (content, completed) VALUES (?, false)";
+            p = c.prepareStatement(query);
+            p.setString(1, content);
+            p.executeUpdate();
+        } finally {
+            DBUtil.close(p,c);
+        }
+        return true;
+    }
```

Comment:
```suggestion
    public Long createTodo(String content) throws SQLException {
        Long id = -1;
        Connection c = null;
        PreparedStatement p = null;
        ResultSet rs = null;

        try {
            c = DBUtil.getConnection();
            String query = "INSERT INTO todos (content, completed) VALUES (?, false)";
            p = c.prepareStatement(query);
            p.setString(1, content);
            p.executeUpdate();
            
            rs = p.getGeneratedKeys();
            if (rs.next()) {
                id = rs.getInt("id");
            }
        } finally {
            DBUtil.close(rs, p,c);
        }
        return id;
    }
```
참고: https://narup.tistory.com/82

insert문 실행 시 auto_increment로 자동 생성되는 id 값 가져오는 방법입니다.